.github/workflows/compatibility.yml

```diff
@@ -12,13 +12,13 @@ jobs:
       max-parallel: 10
       matrix:
         # Minimum support version, ..., Latest version, `beta`.
-        woocommerce: [ '5.4.0', '5.5.0', '5.6.0', '5.7.0', '5.8.0', '5.9.0', '6.0.1', '6.1.2', '6.2.2', '6.3.1', '6.4.1', '6.5.1', 'beta' ]
+        woocommerce: [ '5.6.2', '5.7.2', '5.8.1', '5.9.1', '6.0.1', '6.1.2', '6.2.2', '6.3.1', '6.4.1', '6.5.1', 'beta' ]
```

Comment:
We used to set the minor version to 0 did something change? I believe that using the latest minor makes more sense as we avoid bugs in WooCommerce though.

Reply:
It seemed inconsistent to me to be using `x.y.0` for some of the versions (e.g. `5.9.0`) but the latest `x.y.z` for others (e.g. `6.0.1`), so I standardized on the latest `x.y.z` for all versions here. Happy to change it back if it's an issue, though.